web_widget_open_tab/static/src/js/open_tab_widget.esm.js

```diff
@@ -13,7 +13,7 @@ export class OpenTabWidget extends Component {
         var url = window.location.href;
         var searchParams = new URLSearchParams(url.split("#")[1]);
         searchParams.set("view_type", "form");
-        searchParams.set("id", this.props.value);
+        searchParams.set("id", this.props.record.data.id);
```

Comment:
That might be the problem with the change suggested. By doing this, this will only work with id field